packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.jsx

```diff
@@ -79,13 +79,18 @@ const DigitForm = ({ is_enabled, setEnabled }) => {
                                 'two-factor__button--success': is_success,
                             })}
                             type='submit'
-                            is_disabled={isSubmitting || !isValid}
-                            has_effect
+                            is_disabled={isSubmitting || !isValid || !dirty}
+                            has_effects
```

Comment:
Why `has_effects`? there is no such props for Button component.

Reply:
It was there initially... so i didnt disturb it. Will check on it

Reply:
Done 😂